openmetadata-ui/src/main/resources/ui/cypress/common/AlertUtils.ts

```diff
@@ -370,7 +370,10 @@ export const verifyAlertDetails = (
       // Check Destination type
       cy.get(
         `[data-testid="destination-${destination.category}"] [data-testid="destination-type"]`
-      ).should('contain', startCase(destination.type));
+      ).should(
+        'contain',
+        destination.type === 'Generic' ? 'Webhook' : startCase(destination.type)
```

Comment:
if the backend is giving you "Generic" then we need to change it in the backend and not have conditions like these in the UI cc @mohityadav766